examples/tfa.py

```diff
@@ -37,115 +32,49 @@
-  Path(_output_dir).mkdir(parents=True, exist_ok=True)
-
-  for scenario_id, expert_trajectories in expert_trajectories.items():
-    filename = os.path.join(_output_dir, f'{scenario_id}.jblb')
-    joblib.dump(expert_trajectories, filename)
 
 def run_configuration(argv):
-  """Main
-
-  Args:
-      argv: The commandline argumends
-
-  Raises:
-      ValueError: If the given agent is not sac or ppo
-  """
-  output_dir = os.path.expanduser(FLAGS.output_dir)
-  if output_dir == default_output_dir:
-    output_dir = os.path.join(os.path.expanduser(FLAGS.output_dir), FLAGS.agent, FLAGS.blueprint)
-
-  params = ParameterServer(filename="examples/example_params/tfa_params.json")
-  # params = ParameterServer()
-  params["ML"]["BehaviorTFAAgents"]["CheckpointPath"] = output_dir
-  params["ML"]["TFARunner"]["SummaryPath"] = output_dir
+  # params = ParameterServer(filename="examples/example_params/tfa_params.json")
+  params = ParameterServer()
+  # NOTE: Modify these paths in order to save the checkpoints and summaries
+  params["ML"]["BehaviorTFAAgents"]["CheckpointPath"] = "/Users/hart/Development/bark-ml/checkpoints/"
+  params["ML"]["TFARunner"]["SummaryPath"] = "/Users/hart/Development/bark-ml/summaries/"
   params["World"]["remove_agents_out_of_map"] = True
 
-
   # create environment
-  if FLAGS.blueprint == 'merging':
-    bp = ContinuousMergingBlueprint(params,
-                                    number_of_senarios=2500,
-                                    random_seed=0)
-  elif FLAGS.blueprint == 'intersection':
-    bp = ContinuousIntersectionBlueprint(params,
-                                    number_of_senarios=2500,
-                                    random_seed=0)
-  elif FLAGS.blueprint == 'highway':
-    bp = ContinuousHighwayBlueprint(params,
-                                    number_of_senarios=2500,
-                                    random_seed=0)
-  else:
-    raise ValueError(f'{FLAGS.blueprint} is no valid blueprint. See help.')
-
+  bp = ContinuousMergingBlueprint(params,
+                                  number_of_senarios=2500,
+                                  random_seed=0)
   env = SingleAgentRuntime(blueprint=bp,
-                          render=False)
-
-
-  if FLAGS.agent == 'ppo':
-    ppo_agent = BehaviorPPOAgent(environment=env,
-                                 params=params)
-    env.ml_behavior = ppo_agent
-    runner = PPORunner(params=params,
-                       environment=env,
-                       agent=ppo_agent)
-  elif FLAGS.agent == 'sac':
-    sac_agent = BehaviorSACAgent(environment=env,
-                                params=params)
-    env.ml_behavior = sac_agent
-    runner = SACRunner(params=params,
-                      environment=env,
-                      agent=sac_agent)
-  else:
-    raise ValueError(f'{FLAGS.agent} is no valid agent. See help.')
-
-
+                           render=False)
+
+  # PPO-agent
+  # ppo_agent = BehaviorPPOAgent(environment=env,
```

Comment:
Why leaving commended code in a delivery code? Is it supposed to be used in the future? If so, why not using a flag or a bool config?

Reply:
The examples/tfa.py file is just reverted to the state it was created before our fork was made. Tobias requested this. So all code in here is not relevant for us :+1: